akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/BehaviorSetup.scala

```diff
@@ -66,11 +72,16 @@ private[akka] final class BehaviorSetup[C, E, S](
   def selfClassic: ClassicActorRef = context.self.toClassic
 
   private var mdcPhase = PersistenceMdc.Initializing
+  
+  private val internalLogger = 
+    LoggerFactory.getLogger(context.log.getName + "Internal") 
+  
   def log: Logger = {
     // MDC is cleared (if used) from aroundReceive in ActorAdapter after processing each message,
     // but important to call `context.log` to mark MDC as used
     PersistenceMdc.setMdc(persistenceId, mdcPhase)
     context.log
+    internalLogger
```

Comment:
Previous line is a no-op now?

Shouldn't this entire method be `internalLogger` now to avoid confusion/mixup with the user logger?

Reply:
`context.log` initializes MDC, which we want to be included also for the internalLogger.
A comment about that would be good.